timm/models/efficientvit_mit.py

```diff
@@ -468,7 +648,7 @@ def __init__(
         self.classifier = nn.Sequential(
             nn.Linear(widths[0], widths[1], bias=False),
             nn.LayerNorm(widths[1]),
-            act_layer(inplace=True),
+            act_layer(inplace=True) if act_layer is not nn.GELU else act_layer(approximate="tanh"),
```

Comment:
there's a module and function with an inplace arg that's ignored for GELU and tanh approx https://github.com/huggingface/pytorch-image-models/blob/a38c2b4475959854bcd6cf992312fa9812135816/timm/layers/activations.py#L152